resources/views/menu_links/form/_entry.blade.php

```diff
@@ -110,6 +110,14 @@
             @endif
         </div>
     </div>
+    <div class="col-md-6">
+        {!! Form::hidden('is_active', 0) !!}
+        <label>
+            {!! Form::checkbox('is_active', 1, $model->is_active) !!}
+            {!! __('menu_links.fields.active') !!}
+            <i class="fa-solid fa-question-circle" data-toggle="tooltip" title="{{ __('menu_links.helpers.active') }}"></i>
```

Comment:
Missing the helper block for 40% of users on mobile devices :) the question-circle needs to be hidden on mobile devices too.